pkg/subnet/local.go

```diff
@@ -81,11 +82,12 @@ type setDefaultSnapshotFunc func(string, bool, string, bool) (bool, error)
 type ICMSpec struct {
 	SkipICMDeploy                bool
 	SkipRelayerDeploy            bool
-	Version                      string
+	ICMVersion                   string
```

Comment:
nit: not sure that ICMVersion is a good alternative to Version in `ICMSPec` struct. pls ignore this comment